mne/io/edf/edf.py

```diff
@@ -233,14 +246,18 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     stim[n_start:n_stop] += evid
                 data[stim_channel_idx, :] = stim[start:stop]
             else:
-                # Allows support for up to 17-bit trigger values (2 ** 17 - 1)
                 stim = np.bitwise_and(data[stim_channel_idx].astype(int),
-                                      131071)
+                                      2**17 - 1)
+
+                if biosemi:
+                    stim = stim >> 8  # not sure why but it works
```

Comment:
I think I'd rather put this in a `Notes` section about dealing with `BioSemi` data than modify users' data.